mission_control/navigator_missions/nodes/graph_validations/Missions.gv

```diff
@@ -0,0 +1,31 @@
+// The Round Table
```

Comment:
We can if you prefer. I think it would be nice for everyone to be able to see how the missions are set up.